docker/alpine/Dockerfile

```
@@ -69,18 +69,19 @@ RUN  apk update \
 WORKDIR /root/agent
 
 # bring in the repo contents, minus .dockerignore files
-COPY . .
+COPY . cppagent
 
 ARG WITH_TESTS=false
 ARG WITH_TESTS_ARG=argument
 
-# Build and optionally test
+# Build and optionally test. Unpack the dist to reduce overhead in the following release step.
 RUN  if [ -z "$WITH_TESTS" ] || [ "$WITH_TESTS" = "false" ]; then \
        WITH_TESTS_ARG="--test-folder="; \
      else \
        WITH_TESTS_ARG=""; \
      fi \
   && conan profile detect \
+  && cd cppagent \
   && conan create . \
```

I believe we could eliminate changing directory by replacing `.` with an absolute path.

```suggestion
  && cd cppagent \
  && conan create /root/agent/cppagent \
```